app/src/main/java/com/asfoundation/wallet/support/AlarmManagerBroadcastReceiver.kt

```diff
@@ -38,7 +38,15 @@ class AlarmManagerBroadcastReceiver : BroadcastReceiver() {
       val intent = Intent(context, AlarmManagerBroadcastReceiver::class.java)
 
       val pendingIntent =
-          PendingIntent.getBroadcast(context, 0, intent, PendingIntent.FLAG_CANCEL_CURRENT)
+          PendingIntent.getBroadcast(
+            context,
+            0,
+            intent,
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M)
+              PendingIntent.FLAG_UPDATE_CURRENT or PendingIntent.FLAG_IMMUTABLE
+            else
+              PendingIntent.FLAG_UPDATE_CURRENT
```

Comment:
There was `PendingIntent.FLAG_CANCEL_CURRENT` before, not `PendingIntent.FLAG_UPDATE_CURRENT` 🤔 
Is there a good reason for this change?